truefoundry.yaml

```diff
@@ -401,3 +412,51 @@ components:
         persistence:
           size: 5Gi
       architecture: standalone
+
+  # Whisper
+  - name: cas-whisper
+    env:
+      WHISPER_PORT: 8000
+      WHISPER_MODEL: Systran/faster-distil-whisper-large-v3
+    type: service
+    image:
+      type: image
+      image_uri: fedirz/faster-whisper-server:latest-cpu
+    ports:
+      - host: cas-whisper.truefoundry.com
```

Comment:
How was this host decided?
actually we should not even be exposing this, as we don't intend to hit it directly

Please set `expose: false`

Reply:
Done,
urls are currently determined based on where we are deploying this yaml was for prod cluster. 
We are in process of generating a `deploy.py` file, which should take `workspace_fqn` and cluster `base_domain_url` as args, this should solve the issue soon.